src/apps/persistence/app_metadata.clj

```diff
@@ -790,7 +811,7 @@
                   [:jt.name           :job_type]
                   [:jt.system_id      :system_id]
                   [:t.external_app_id :external_app_id])
-          (where {:a.id (uuidify app-id)})
+          (where {:v.id (-> app-id uuidify get-app-latest-version)})
```

Comment:
This looks like one of those areas that will change in a later update, since it's always grabbing the latest version of the app. Is that correct?

Reply:
Yes, correct.

I figure once I add an endpoint that wants this info for a specific version, I'll update this function to take an `app-version-id`, then the existing endpoints (which will always want info for the latest version) will be updated to lookup the latest version ID and pass it to this function.